src/lib/ownership/OwnershipEngine.ts

```diff
@@ -19,7 +19,7 @@ export class OwnershipEngine {
     const matchers = [...this.matchers].reverse();
 
     for (const matcher of matchers) {
-      if (matcher.match(filePath)) {
+      if (filePath.includes(matcher.path)) {
```

Comment:
The purpose of this change is to match the first `matcher.path` included in the filePath that is being analyzed. 
In our `CODEOWNERS` file we have something like this:
```
web/ @yankaindustries/pod-core-frameworks-web
web/apps/core/ @yankaindustries/team-consumer
```
When running the `audit` command, the output was that `@yankaindustries/pod-core-frameworks-web` owned everything under `web/`
![image](https://user-images.githubusercontent.com/24876897/171880805-e3c019f4-f4da-4ee4-b81f-ee2ec7ac5fad.png)

With these changes, we are able to match at first the subdirectories and then continue with their parent folder. The output is the expected:
![image](https://user-images.githubusercontent.com/24876897/171881322-569e6fcc-9e22-4cbc-87b9-2c5ad893e487.png)